scopes/api-reference/renderers/api-node-details/api-node-details.module.scss

```diff
@@ -30,13 +30,13 @@
   > section {
     overflow: hidden;
     border-radius: 8px;
+    background-color: var(--bit-bg-heavy, #f4f5f6);
+    border: 1px solid var(--bit-bg-heaviest, #eeeff2);
+    width: unset !important;
```

Comment:
to override monaco?
if so its ok but please leave a comment why this is needed.
`// override monaco width to prevent overflowing from...`